recipe/meta.yaml

```diff
@@ -34,17 +34,18 @@ requirements:
     - pip
     - setuptools
     - cython >=0.24.0
-    - cudnn
     - fastrlock >=0.3
+    - cudnn
     - nccl
 
   run:
     - python
     - setuptools
-    - cudnn
     - fastrlock >=0.3
     - numpy >=1.9.0
     - six >=1.9.0
+    # - cudnn  <-- added via run_export
```

Comment:
Before we do this, can you please submit a PR to conda-forge-pinning to update [this line to use `7.6.4`]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/73abc83abf4fb35f8030413c35916364103e64be/recipe/conda_build_config.yaml#L360-L361 )?

We will then need a new build of conda-forge-pinning. Then we can re-render here and take advantage of `run_exports`.

ref: https://github.com/AnacondaRecipes/aggregate/issues/173#issuecomment-550072394

Reply:
Aren't we already effectively pinned at 7.6.4 given what @henryiii found [here](https://github.com/conda-forge/cupy-feedstock/pull/16#issuecomment-562430466)?

Reply:
Not really. An arbitrary variant like [this one sets the pin here]( https://github.com/conda-forge/cupy-feedstock/blob/1ab88f84e3e473d89b452b12e8f1146adbcf48b7/.ci_support/linux_cuda_compiler_version10.0python2.7.yaml#L13-L14 ). However it is just pulling this information [from conda-forge-pinning]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/73abc83abf4fb35f8030413c35916364103e64be/recipe/conda_build_config.yaml#L360-L361 ), which is where we set our pins at a global level. So we need to update `conda-forge-pinning` to have a pin that includes the version with the `run_export`.

Reply:
What if we do this:
```
  host:
    - cudnn == 7.6.4
```
Would this work as a valid pinning?

Reply:
Yeah that would be equivalent. The other option would be to `conda_build_config.yaml` (basically like `conda-forge-pinning`, but only for our pins).

Better still might be...

```yaml
  host:
    - cudnn >=7.6.4
```

This would ensure we just have some version of `cudnn` that has `run_exports` (preferably newer ones).

Reply:
The main thing is we want to make sure we do get a version of `cudnn` that does have `run_exports`.

Reply:
Sounds good. Let us take this approach. Once `conda-forge-pinning` gets updated we can revisit this when the next release is out (or whenever we want). I hate creating PR dependencies. It makes me nervous.